docs/development/Debugging_Pipeline.md

```diff
@@ -0,0 +1,34 @@
+# Debugging QE Pipeline Failures
+
+When QE pipeline fails, the console output will display a ssh address
+such as:
+
+    ssh <REDACTED>@nyc1.tmate.io
+
+This can be used to get an active session on the MacOS host the pipelines
+run on. This creates a tmux session as `root` on the MacOS host. From
+here, switch to the `runner` user:
+
+    # sudo su runner
+    $ cd
+
+In `$HOME/work/pki/pki/` is the cloned GitHub repo, plus Vagrant home. There
+are two VMs currently provisioned in this pipeline:
+
+    - `controller` at IP address `192.168.33.10`, which runs Ansible
+    - `master` at IP address `192.168.33.20`, which runs PKI (and is
```

Comment:
Markdown considers this as a code block. Can you remove the spaces before the `-`?